scalatest/src/main/scala/org/scalatest/tools/ScalaTestFramework.scala

```diff
@@ -315,8 +316,8 @@ class ScalaTestFramework extends SbtFramework {
    * Returns an <code>org.scalatools.testing.Runner</code> that will load test classes via the passed <code>testLoader</code>
    * and direct output from running the tests to the passed array of <code>Logger</code>s.
    */
-  def testRunner(testLoader: ClassLoader, loggers: Array[Logger]) = {
-    new ScalaTestRunner(testLoader, loggers)
+  def testRunner(testLoader: ClassLoader | Null, loggers: Array[Logger | Null] | Null) = {
+    new ScalaTestRunner(testLoader.nn, loggers.nn.map(_.nn))
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.